### DISCUSSION TITLE
Support for VSCode markdown and vue syntax

### DISCUSSION THREAD

**Discussion by melishev** [vuejs/vitepress#4304] [Q&A] [answered]
This may be a settled question, but I haven't found an answer.

When I use vue syntax in markdown files, I would like vscode to understand what I am writing and prompt me as if it were a .vue file. Can anyone suggest a setting for this?

**Comment by brc-dd** [comment, marked as answer]
https://vitepress.dev/guide/using-vue#vs-code-intellisense-support

**Comment by amaanx86** [comment]
you can use markdown extension
